qe_metrics/utils/issue_utils.py

```diff
@@ -113,3 +113,18 @@ def create_update_issues(
         current_issues=issues, db_issues=JiraIssuesEntity.select(product=product, severity=severity), product=product
     )
     orm.commit()
+
+
+def delete_old_issues(days_old: int) -> None:
+    """
+    Delete issues from the database that were last updated more than the number of days defined in days_old.
+
+    Args:
+        days_old (int): Number of days from the last_updated date to keep issues in the database
+    """
+    issues = JiraIssuesEntity.select(  # noqa: FCN001
+        lambda _issue: _issue.last_updated < (datetime.now().date() - timedelta(days=days_old))
+    )
+    LOGGER.info(f"Deleting {len(issues)} issues that haven't been updated in {days_old} days from the database")
+    [issue.delete() for issue in issues]
```

Comment:
can we get exception here? maybe we should be in the safe side?

we can use https://github.com/RedHatQE/pyhelper-utils/blob/b4749bfaf92e7c419f8ed4113042ddd35c50e689/pyhelper_utils/general.py#L42

Reply:
@myakove let me know if that isn't what you are looking for.